src/main/java/gregtech/api/pattern/TraceabilityPredicate.java

```diff
@@ -40,7 +41,18 @@ public class TraceabilityPredicate {
             return true;
         }
         return false;
-    }, () -> GregTechAPI.HEATING_COILS.keySet().stream().map(key -> new BlockInfo(key, null)).toArray(BlockInfo[]::new))
+    }, () -> GregTechAPI.HEATING_COILS.entrySet().stream()
+            .filter(entry -> {
+                // do not include the active versions of blocks in jei to prevent "duplicates"
+                if (entry.getKey().getPropertyKeys().contains(VariantActiveBlock.ACTIVE)) {
+                    return !entry.getKey().getValue(VariantActiveBlock.ACTIVE);
+                }
+                return true;
+            })
```

Comment:
Since you're using this identical rather verbose predicate in two places, maybe it would make sense to extract it for reuse?